CAM/CreateCamera.md

```diff
@@ -7,25 +7,17 @@ ns: CAM
 // 0x5E3CF89C6BCCA67D 0x5D6739AE
 Cam CREATE_CAMERA(Hash camHash, BOOL active);
 ```
-Creates a camera with the specified camera hash, You can use `SET_CAM_` natives to manipulate the camera.  
+Creates a camera with the specified camera hash, You can use `SET_CAM_` natives to manipulate the camera. 
 Make sure to call [RENDER_SCRIPT_CAMS](#_0x07E5B515DB0636FC) once the camera is created, or this won't have any visible effect.
 
+Take a look at [CREATE_CAM](#_0xC3981DCE61D9E13F) if you would like to see the available camera names.
+
 ```
 NativeDB Introduced: v323
 ```
 
-### Camera hashes:
-- 26379945 for DEFAULT_SCRIPTED_CAMERA 
-- 964613260 for DEFAULT_ANIMATED_CAMERA
-- 180543640 for DEFAULT_SPLINE_CAMERA
-- -2001091 for DEFAULT_SCRIPTED_FLY_CAMERA
-- 1775630800 for TIMED_SPLINE_CAMERA
-- 1665938388 for CUSTOM_TIMED_SPLINE_CAMERA
-- 457439121 for ROUNDED_SPLINE_CAMERA
-- -119532588 for SMOOTHED_SPLINE_CAMERA
-
 ## Parameters
-* **camHash**: The hash of the camera type
+* **camHash**: The hash of the camera type, use [GET_HASH_KEY](_0xD24D37CC275948CC) to get the camera hash from the name
```

Comment:
Hey, you have a missing hash i.e. `#_0xD24D37CC275948CC`, please add it, thanks! 🙂